### PR TITLE
create playlist implmented with integration test

### PR DESCRIPTION
**Pull request by nagarjunaboddu**
Please review and let merge.

### CODE REVIEW

File: src/main/java/com/galvanize/playlistservice/entities/Playlist.java [hunk 1]
```diff
@@ -10,6 +10,41 @@
...
+        return songs;
+    }
+
+    public void setSongs(List<Song> songs) {
+        this.songs = songs;
+    }
+
     @ElementCollection
     private List<Song> songs;
+
```
Comment (onanna) [inline src/main/java/com/galvanize/playlistservice/entities/Playlist.java]: please can you move this up

Comment (onanna) [reply src/main/java/com/galvanize/playlistservice/entities/Playlist.java]: songs property up line 45 and 46

**Review by onanna — APPROVED** [review]
LGTM

**Review by bharathi1988 — APPROVED** [review]
Review Completed

**Pull request merged**